draft-ietf-httpbis-layered-cookies.md

```diff
@@ -980,13 +1029,15 @@ A host _host_ **Domain-Matches** a string _domainAttributeValue_ if at least one
 To determine the **Cookie Default Path**, given a URL path _path_, run these steps.
 They return a URL path.
 
-1.  Assert: _path_ is a non-empty list.
+1.  Assert: _path_ is non-empty and starts with 0x2f (/).
+
+2.  Let _splitPath_ be _path_ split on 0x2F (/).
 
-2.  If _path_'s size is greater than 1, then remove _path_'s last item.
+3.  If _splitPath_'s size is greater than 1, then remove _splitPath_'s last item.
 
-3.  Otherwise, set _path_[0] to the empty string.
+4.  Otherwise, set _splitPath_[0] to the empty string.
 
-4.  Return _path_.
+5.  Return _splitPath_.
```

Comment:
This is all wrong. A URL path is defined as a list.

Reply:
I did not know that. I'll revert